deploy/aws/lambda-ext/main.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021-2025 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/cerbos/cerbos/internal/config"
+	"github.com/cerbos/cerbos/internal/observability/logging"
+	runutils "github.com/cerbos/cerbos/internal/run"
+	"github.com/cerbos/cerbos/internal/server"
+	"github.com/cerbos/cerbos/pkg/cerbos"
+	"github.com/sourcegraph/conc/pool"
+	"go.uber.org/zap"
+)
+
+func main() {
+	ctx, stopFunc := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
+	defer stopFunc()
+
+	configPath := os.Getenv("CERBOS_CONFIG")
+	if configPath == "" {
+		configPath = "/conf.yml"
```

Comment:
The default config file name we use is `.cerbos.yaml`

```suggestion
		configPath = "/.cerbos.yaml"
```